Portfolio.html

```diff
@@ -4,11 +4,85 @@
     <meta charset="UTF-8">
     <link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
     <link rel="stylesheet" type="text/css" href="Portfolio.css">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type="text/css" href="Media-Query.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Rob Reeves Portfolio</title>
 </head>
-<body>
-
-
 
+<div>
+    <nav class="main-nav">
+        <div class="nav-button-left">
+            <a class="nav-text" href="#C1">ABOUT ME</a>
+            <a class="nav-text" href="#C3">CONTACT</a>
+        </div>
+        <div class="nav-button-right">
+            <a class="nav-text" href="#C2">THREE PEDALS ONLY</a>
+        </div>
+    </nav>
+        <div class="main-header-wrapper">
```

Comment:
Consider using a header tag in place of a div tag

Reply:
done.